### ISSUE TITLE
Flag With File Attachment?

### ISSUE THREAD

**Issue by tsull360** [moloch--/RootTheBox#626]
Hello,
How can I create a flag that has an attached file? E.g. a flag with a picture of a location where the challenge is to identify where the picture was taken.

**Comment by eljeffeg** [comment]
Take a look at the [Game Materials](https://github.com/moloch--/RootTheBox/wiki/Game-Setup-Tutorial#game-materials)

**Comment by tsull360** [comment]
Gotcha, so add the file to game materials, and then create a url link to it as a part of the flag?

**Comment by eljeffeg** [comment]
Yeah or you can reference that the file is in the game materials as it's a menu option the user can browse.  You could also host the file offsite at a url of your choice and link to that.
